server/src/test/java/org/elasticsearch/index/analysis/CustomNormalizerTests.java

```diff
@@ -31,11 +30,8 @@
 
 import java.io.IOException;
 import java.io.Reader;
-import java.util.List;
 import java.util.Map;
-import java.util.function.Function;
 
-import static java.util.Collections.singletonList;
 import static java.util.Collections.singletonMap;
 
 public class CustomNormalizerTests extends ESTokenStreamTestCase {
```

Comment:
Do we still need this test when CustomNormalizerProvider is gone? I saw there is also no CustomAnalyzerTests, maybe this could be renamed and a few tests added that test the CustomAnalyzer parts that are not covered here?

Reply:
There's still a distinction in settings between analyzers and normalizers, so I think this test is still useful?